ykrt/src/compile/jitc_yk/jit_ir/mod.rs

```diff
@@ -415,6 +415,16 @@ impl Module {
         self.insts[usize::from(iidx)] = inst;
     }
 
+    /// Replace the instruction it `iidx` with an instruction that will generate `op`. In other
```

Comment:
s/it/in

Reply:
Fixed in 762413f260011101f75734f14ca64702da0bc210.